python/ray/train/tests/test_torch_trainer.py

```diff
@@ -111,6 +129,58 @@ def test_checkpoint_freq(ray_start_4_cpus):
         trainer.fit()
 
 
+@pytest.mark.parametrize("num_gpus_per_worker", [0.5, 1, 2])
+def test_tune_torch_get_device_gpu(ray_2_node_4_gpu, num_gpus_per_worker):
```

Comment:
Should we make this ray_2_node_2_gpu? Otherwise, if we have more GPUs in the cluster than workers, then placement can be non-deterministic?

Reply:
changed to `ray_2_node_2_gpu `